test/functional/core/job_spec.lua

```diff
@@ -23,6 +24,9 @@ local expect_msg_seq = helpers.expect_msg_seq
 local pcall_err = helpers.pcall_err
 local Screen = require('test.functional.ui.screen')
 
+-- FIXME(dundar): Flaky set of tests. I suspect the before_each is faulty and that it infects all
```

Comment:
hmm, clear() should take care of that

Reply:
It's possible that `before_each()` hangs